### PR TITLE
sncosmo

### PR DESCRIPTION
**Pull request by kbarbary**
Unfortunately I didn't include a license file in the manifest when releasing to pypi for the last version; will do so for the next version.

### CODE REVIEW

**Comment by kbarbary** [comment]
The build for this package requires the `astropy-helpers` package, which is available from the openastronomy conda channel. Is there a way to specify that channel in the build somehow?

**Comment by dopplershift** [comment]
No, `astropy-helpers` will need to be added to conda-forge as well.

File: recipes/sncosmo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (astrofrog) [inline recipes/sncosmo/meta.yaml]: You don't actually need astropy-helpers, you can just include here ``--offline --no-git``

Comment (astrofrog) [inline recipes/sncosmo/meta.yaml]: Ah yes indeed, sorry!

**Comment by kbarbary** [comment]
sncosmo no longer requires astropy-helpers at build time, so this should be all set now. cc @mwcraig

**Comment by mwcraig** [comment]
Thanks, @kbarbary...next up, try out some channel copying!

**Pull request merged**